### PR TITLE
i18n(zh-tw): Update editor-setup.mdx

### PR DESCRIPTION
**Pull request by CheukTsai**
#### Description (required)

Update editor-setup.mdx

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/zh-tw/editor-setup.mdx [hunk 1]
```diff
@@ -54,15 +57,37 @@ JetBrains 將來的 [Fleet IDE](https://www.jetbrains.com/fleet/) _將會_ 支
...
+### Stylelint
+
+[Stylelint](https://stylelint.io/) 是一種流行的 CSS linter。[社群維護的 Stylelint 配置](https://github.com/ota-meshi/stylelint-config-html) 提供 Astro 支援。
+
+安裝說明、編輯器整合和其他資訊可以在專案的 README 文件中找到。
+
 ### Prettier
 
-[Prettier](https://prettier.io/) 是個有名的格式化工具，支援了 JavaScript, HTML, CSS 等。要加入對 `.astro` 檔的格式化支援可以使用[官方的 Astro Prettier 外掛](https://github.com/withastro/prettier-plugin-astro)。
+[Prettier](https://prettier.io/) 是個有名的格式化工具，支援了 JavaScript, HTML, CSS 等。[Prettier](https://prettier.io/) 是一種流行的 JavaScript、HTML、CSS 等格式化程式。 如果您使用 [Astro VS Code 擴充功能](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) 或 [其他編輯器中的 Astro 語言伺服器](#other-code - editors)，包括了 Prettier 程式碼格式化功能。
```
Comment (VoxelMC) [inline src/content/docs/zh-tw/editor-setup.mdx:68]: I believe that this sentence is doubled!
```suggestion
[Prettier](https://prettier.io/) 是一種流行的 JavaScript、HTML、CSS 等格式化程式。 如果您使用 [Astro VS Code 擴充功能](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) 或 [其他編輯器中的 Astro 語言伺服器](#other-code - editors)，包括了 Prettier 程式碼格式化功能。
```

**Review by VoxelMC — COMMENTED** [review]
Congrats on your first translation PR!
I have added a comment for something that might need to be fixed!

Other than that comment, once it is corrected I can say...
`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

**Comment by CheukTsai** [comment]
Thank you for your comment @VoxelMC! I accidentally doubled it with the previous translation of the same sentence there. I have committed your suggestion there :)

**Review by liruifengv — APPROVED** [review]
LGTM. Thanks for your contrbution. 

if you have more PR in the future, I'm here to help.

**Review by VoxelMC — APPROVED** [review]
`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

**Review by Yan-Thomas — APPROVED** [review]
LGTM, thanks!

**Pull request merged**